src/expressions/StaticContext.ts

```diff
@@ -37,6 +37,7 @@ export type FunctionDefinition = {
  */
 export default class StaticContext {
 	public parentContext: IContext;
+	public registeredDefaultFunctionNamespace: string = 'http://www.w3.org/2005/xpath-functions';
```

Comment:
agreed, but this should've been part of the default function namespace Pr

Reply:
Might as well take it along in this PR as we now see it. Otherwise it'll likely be forgotten.
It would also be fine if we fix this directly on master as it should indeed have been fixed in another PR but was overlooked there.